packages/vuetify/src/locale/zh-Hant.ts

```diff
@@ -1,40 +1,40 @@
 export default {
-  close: 'Close',
+  close: '關閉',
   dataIterator: {
-    pageText: '{0}-{1} 共 {2} 條',
-    noResultsText: '沒有找到匹配記錄',
-    loadingText: 'Loading items...',
+    pageText: '{2} 條中的 {0}~{1} 條',
+    noResultsText: '沒有符合條件的結果',
+    loadingText: '讀取中...',
   },
   dataTable: {
-    itemsPerPageText: '每頁行數：',
+    itemsPerPageText: '每頁列數：',
     ariaLabel: {
-      sortDescending: ': Sorted descending. Activate to remove sorting.',
-      sortAscending: ': Sorted ascending. Activate to sort descending.',
-      sortNone: ': Not sorted. Activate to sort ascending.',
+      sortDescending: '：降序排列。點擊以移除排序方式。',
+      sortAscending: '：升序排列。點擊以移除排序方式。',
```

Comment:
The original text says 'Activate to sort descending' but the translation has given it a different meaning.

Reply:
If you got a proper translation let me know what it should be and I can patch it in (of you can do a quick PR)